lib/room.rb

```diff
@@ -1,12 +1,23 @@
-class Rom
+class Room
+  attr_reader :category
 
-  def initialize(category, length, width)
-    @category = category
-    @length = 10
-    @width = width
+  def initialize(category_parameter, length_parameter, width_parameter)
+    @category = category_parameter
+    @length = length_parameter.to_i
+    @width = width_parameter.to_i
+    @painted = false
+    @area = 0
```

Comment:
No need for this to be an instance variable. This can just be calculated by the method when the method is called.  It's best practice to avoid creating instance variables unless we really need them.